extensions/wikia/AdEngine/js/template/porvata.js

```diff
@@ -126,6 +131,22 @@ define('ext.wikia.adEngine.template.porvata', [
 			}
 		});
 
+		if (params.slotNameForFallbackBid) {
+			video.addEventListener('wikiaEmptyAd', function () {
+				var fallbackBid = prebid.getWinningVideoBidBySlotName(params.slotNameForFallbackBid, fallbackBidders);
```

Comment:
can we avoid calling this method when we know we won't use this bid? (`fallbackAdRequested = true`)